.github/workflows/specs.yml

```diff
@@ -3,14 +3,26 @@ on: [push, pull_request]
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref_name != 'master' }}
+
 jobs:
-  tests:
+  spec:
     strategy:
       fail-fast: false
       matrix:
         os:
           - ubuntu-latest
           - macos-13
+        test:
+          - "tests/fast/*"
+          - "tests/long/truffleruby_comment_test.sh"
+        include:
+          - os: ubuntu-latest
+            test: "tests/long/named_ruby_and_gemsets_comment_test.sh"
+          - os: ubuntu-latest
+            test: "tests/long/ruby_prepare_mount_comment_test.sh"
```

Comment:
Could you preserve the comments for these 2 jobs (as to why they are not run on macOS)?